modules/otel-about-rn.adoc

```diff
@@ -6,7 +6,7 @@
 [id="otel-product-overview_{context}"]
 = About this release
 
-{OTELName} 3.7 is provided through the link:https://catalog.redhat.com/software/containers/rhosdt/opentelemetry-operator-bundle/615618406feffc5384e84400/history[{OTELOperator} 0.135.0] and based on the open source link:https://opentelemetry.io/docs/collector/[OpenTelemetry] release 0.135.0.
+{OTELName} 3.8 is provided through the link:https://catalog.redhat.com/software/containers/rhosdt/opentelemetry-operator-bundle/615618406feffc5384e84400/history[{OTELOperator} 0.???.0] and based on the open source link:https://opentelemetry.io/docs/collector/[OpenTelemetry] release 0.???.0.
```

Comment:
The use of the `?` characters is intended here. I am waiting for Engineering to finalize these digits.